IncludeToolbox/IncludeFormatter/IncludeLineInfo.cs

```diff
@@ -92,6 +92,8 @@ public static IncludeLineInfo[] ParseIncludes(string text, bool removeEmptyLines
                 if (multiLineCommentStart > -1 && multiLineCommentStart < includeOccurence) // Multi line comment started in same line, but before #include.
                     continue;
 
+                if (lines[line].Contains(".inl>") || lines[line].Contains(".inl\"")) // Don't remove inline files.
```

Comment:
Do you ever want to remove "inline" files?   The ending is problematic, though.
.inc is probably used..?   A list of: "skip these file endings" might be cleanest; yes.